pipeline_template.yaml

```diff
@@ -0,0 +1,37 @@
+---
+version:
+
+date:
+
+pipelines:
+  - models:
+    model_name:
+    pipeline_name:
+    created_by:
+    creation_date:
+    version:
+    description:
+    job_name:
+    dataset:
+    metrics:
+    identifiable:
+    default:
+
+models:
+  - tasks:
+    api_call_function:
+    endpoint:
+    api_key:
+    inference_function:
+    content_type:
+    deployment_platform:
```

Comment:
is this necessary?

Reply:
Yes, it is necessary to build the `headers` for the HTTP requests.

https://github.com/ai-cfia/nachet-backend/blob/0b4e0a4a018040d012fd321ab01ae44e2197927d/model/seed_detector.py#L79